src/components/read-along-component/read-along.tsx

```diff
@@ -463,7 +507,29 @@ export class ReadAlongComponent {
     intersectionObserver.observe(reading_el)
     this.scrollTo(reading_el)
   }
+//for when you visually align content
+  inParagraphContentOverflow(element: HTMLElement): boolean {
+    let para_el = this._getSentenceContainerOfWord(element);
+    let para_rect = para_el.getBoundingClientRect()
+    let el_rect = element.getBoundingClientRect()
+
+    // element being read is left of the words being viewed
+    let inOverflowLeft = el_rect.right  < para_rect.left;
+    // element being read is right of the words being viewed
+    let inOverflowRight = el_rect.right  > para_rect.right;
 
+    let intersectionObserver = new IntersectionObserver((entries) => {
+      let [entry] = entries;
+      if (entry.isIntersecting) {
+        setTimeout(() => { this.showGuide = false; this.autoScroll = true }, 100)
+        intersectionObserver.unobserve(element)
+      }
+    })
+    intersectionObserver.observe(element)
+//console.log("inParagraphContentOverflow",inOverflowLeft,inOverflowRight,para_rect,el_rect,para_el)
```

Comment:
Please remove any commented out code like debug prints before merging the changes.

```suggestion
```

Reply:
will do